lib/menu.rb

```diff
@@ -0,0 +1,15 @@
+class Menu
+  attr_reader :dishes
+  def initialize
+    @dishes = [
+        { name: 'Burger', price: 10.95  },
+        { name: 'Pizza', price: 14.00 },
+        { name: 'Salad', price: 7.60 },
+        { name: 'fries', price: 2.90 }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.